ccmlib/scylla_node.py

```diff
@@ -1876,6 +1945,41 @@ def run_scylla_types(self, action: str, scylla_type: ScyllaType, *values: Tuple[
             raise ToolError(command=' '.join(cmd_and_args), exit_status=res.returncode, stdout=res.stdout, stderr=res.stderr)
         return res.stdout.strip()
 
+    def run_repair(self, options, timeout=None):
+        return self.nodetool("repair " + options, timeout=timeout)
+
+    def run_cluster_repair(self, options, timeout=None):
+        return self.nodetool("cluster repair " + options, timeout=timeout)
+
+    def repair(self, keyspace: Optional[str] = None, tables: Optional[List[str]] = None, hosts: Optional[List[str]] = None, dcs: Optional[List[str]] = None,
```

Comment:
I do not like naming of this, difference between `run_repair` and `repair` is not easily gueassable and it would be very confusing to me if I read without deeper knowledge of the system. This is not helped by the fact that there are no docstrings.

I would expect `repair` to do a nodetool repair directly, but it is not that and it incorporates both `repair` and `cluster_repair`. I would name it something else to signify it is not a direct call to the server but involves a bit of logic as well. `repair_all` might work?

Reply:
Is it also how the customers are gonna use it? I.e. are they expected to run both repairs?

Reply:
Yes, they do